src/utils/ethers/ethers.js

```diff
@@ -23,16 +22,43 @@ export const getAllMintedNFTs = () => {
             const fromBlock = 4795836;
```

Comment:
Store the contract creation block in the json with the contract addresses.
We will have multiple contracts => multiple creation blocks (from block)